examples/google-signin/google-signin-iframe.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <script src="https://accounts.google.com/gsi/client"></script>
+  <script>
+    // Add all allowed publisher domains.
+    const ALLOWED_ORIGINS = [
+      'http://localhost:8000',
+      'https://scenic-2017.appspot.com/'
+    ];
+    let pendingNonce = null;
+    let domainVerifiedCallback = null;
+
+    function generateRandom() {
+      return btoa((Math.floor(Math.random() * 100000) + 1) + '-nonce');
+    }
+
+    function notifyParent(message) {
+      window.parent && window.parent.postMessage(message, '*');
+    }
+
+    function registerDomainVerifier(done) {
+      if (!window.parent) {
+        return;
+      }
+      domainVerifiedCallback = done;
+      window.addEventListener('message', (event) => {
+          if (event.source != window.parent || !event.data) {
+              return;
+          }
+          if (!pendingNonce || !domainVerifiedCallback) {
+              return;
+          }
+          if (event.data['sentinel'] != 'onetap_google' || event.data['command'] != 'parent_frame_ready') {
+              return;
+          }
+          if (!event.data['nonce'] || event.data['nonce'] != pendingNonce) {
+              return;
+          }
+          pendingNonce = null;
+          if (ALLOWED_ORIGINS.includes(event.origin)) {
+              let callback = /** typeof {function} */ (domainVerifiedCallback);
+              domainVerifiedCallback = null;
+              callback();
+          }
+        });
+    }
+
+    function requestDomainVerification() {
+      pendingNonce = generateRandom();
+      notifyParent({
+        sentinel: 'onetap_google',
+        command: 'intermediate_iframe_ready',
+        nonce: pendingNonce
+      });
+    }
+
+    function formPost(url, data) {
+      const form = /** @type {!HTMLFormElement} */ (document.createElement('form'));
+      document.body.appendChild(form);
+      form.method = 'post';
+      form.action = url;
+      if (data) {
+        Object.keys(data).map(function (name) {
+          let input = /** @type {!HTMLInputElement} */ (
+            document.createElement('input'));
+          input.type = 'hidden';
+          input.name = name;
+          input.value = data[name];
+          form.appendChild(input);
+        });
+      }
+      form.submit();
+    }
+
+    function handleLoginResponse(response) {
+      if (!response || !response['credential']) return;
+      responseReturned = true;
+      notifyParent(response);
```

Comment:
need to call formPost here and get a user data back to pass to parent.

Reply:
use fetch not form so we can get json back.

Reply:
Modified the code to use fetch (with a POST) through the SwG XHR class.

Reply:
Decided on leaving this up to the publisher (but recommended fetch).